src/module-elasticsuite-swatches/Block/Navigation/Renderer/Swatches/RenderLayered.php

```diff
@@ -37,15 +40,20 @@ public function getSwatchData(): array
         }
 
         $attributeOptions = [];
+        // Collect parameter labels in the expected order
```

Comment:
A full stop (".") is required at the end of the sentence to avoid a phpcs error.

Reply:
sorry. my phpcs settings broke. I fixed phpcs and his errors